docs/src/data/hid.js

```diff
@@ -2983,7 +2985,7 @@ export default [
       linux: true,
       android: false,
       macos: null,
-      ios: null,
+      ios: true,
```

Comment:
Are you sure this doesn't need manual testing?

Reply:
nope.  those three escaped my revision.  i appreciate your thoroughness.  i've updated those three keys to `null`